digital-deck-ui/src/components/MainMenu.vue

```diff
@@ -1,15 +1,57 @@
 <template>
```

Comment:
We'll want another page for the lobby. It should show up after a player creates/joins the room. It's like a waiting room so other players can join before the game begins. The host will have the option to start the game whenever they want and then all the players will be moved to the game screen.

Reply:
Okay, I created this page but this is gonna show the same component that we show in Game

Reply:
How do you mean?